modules/programs/k9s.nix

```diff
@@ -34,7 +34,8 @@ in {
       type = yamlFormat.type;
       default = { };
       description = ''
-        Skin written to {file}`$XDG_CONFIG_HOME/k9s/skin.yml`. See
+        Deprecated; see `programs.k9s.skins`.
+        Skin written to {file}`$XDG_CONFIG_HOME/k9s/skins/default.yaml`. See
```

Comment:
Should we set `settings.k9s.ui.skin = "default"` when unset and the fallback is used? Otherwise the default skin won't be used.

Reply:
Definitely. I forgot to implement that.